auth/utils/read-external-sources.ts

```diff
@@ -97,8 +96,31 @@ function filterPropertiesByServiceName(envObj: any, serviceName: string): any {
  * Pulls credentials from VCAP_SERVICES env property that IBM Cloud sets
  *
  */
+function getCredentials(name) {
```

Comment:
After copying this code from its original package into this file, I think it would be good to give this a more descriptive name.  
In fact, I'll suggest that you rename `getCredentialsFromCloud` to be `getPropertiesFromVCAP` and then rename `getCredentials` to be something like `getVCAPCredentialsForService`, or just move the `getCredentials` logic into the newly-renamed `getPropertiesFromVCAP`.